spec.html

```diff
@@ -12480,7 +12480,7 @@ <h1>Ordinary and Exotic Objects Behaviours</h1>
 
   <emu-clause id="sec-ordinary-object-internal-methods-and-internal-slots">
     <h1>Ordinary Object Internal Methods and Internal Slots</h1>
-    <p>All ordinary objects have an internal slot called [[Prototype]]. The value of this internal slot is either *null* or an object and is used for implementing inheritance. Data properties of the [[Prototype]] object are inherited (and visible as properties of the child object) for the purposes of get access, but not for set access. Accessor properties are inherited for both get access and set access.</p>
+    <p>All ordinary objects have an internal slot called [[Prototype]]. The value of this internal slot is either *null* or an object and is used for implementing inheritance. Data properties of the [[Prototype]] object are inherited (and visible as properties of the child object) for get access; for set access, only writability is inherited, with assignment setting properties on the child object if the data property on the [[Prototype]] object is writable. Accessor properties are inherited for both get access and set access.</p>
```

Comment:
```suggestion
    <p>All ordinary objects have an internal slot called [[Prototype]]. The value of this internal slot is either *null* or an object and is used for implementing inheritance. For data property get access, when the property is missing from the child object, the same-named property of the [[Prototype]] object is inherited (and visible as if it was a property of the child object). For data property set access, when the property is missing from the child object, the writability of the same-named property from the [[Prototype]] object is used to determine whether a property is created on the child object or the assignment fails. Accessor properties are inherited for both get access and set access.</p>
```

Reply:
I didn't like this rewording because leading with "For data property get access" read a bit funny to me, like you decide when you do a Get that it's a data property Get vs an accessor Get. I took another stab at rewording in a new commit.